create_builder.go

```diff
@@ -103,6 +103,14 @@ func (c *Client) CreateBuilder(ctx context.Context, opts CreateBuilderOptions) e
 }
 
 func validateBuildpack(bp dist.Buildpack, source, expectedID, expectedBPVersion string) error {
+	if bp == nil {
```

Comment:
What's the use case in which this error would present itself?

Reply:
I guess it can't really happen given the flow where we validate the descriptor and the fact that we start with an empty struct. I've just gotten paranoid about interfaces not getting set, but that seems impossible here.